syntax = "proto3";

package order;

message OrderPlaced {
  string user_name = 1;
  string product_name = 2;
  int32 quantity = 3;
}
